src/analyzer/protocol/smb/smb2-com-create.pac

```diff
@@ -17,6 +36,18 @@ refine connection SMB_Conn += {
 			requestinfo->Assign(0, filename);
 			requestinfo->Assign(1, val_mgr->GetCount(${val.disposition}));
 			requestinfo->Assign(2, val_mgr->GetCount(${val.create_options}));
+			requestinfo->Assign(3, val_mgr->GetCount(${val.access_mask}));
+			requestinfo->Assign(4, val_mgr->GetCount(${val.share_access}));
+
+            		if (${val.contexts})
+                		{
+                		VectorVal* cv = new VectorVal(BifType::Vector::SMB2::CreateContextValues);
+
+		        	for ( auto i = 0u; i < ${val.contexts}->size(); ++i )
+	                		cv->Assign(i, BuildSMB2CreateContextVal(${val.contexts[i]}));
+                		requestinfo->Assign(5, cv);
```

Comment:
The `contexts` field of `SMB2::CreateRequest` isn't `&optional`, so should unconditionally assign it a `VectorVal`.  Whether that value gets populated or remains empty is the logic that should be conditional.